### PR TITLE
Create CI jobs for branch-4.11

### PR DESCRIPTION
**Pull request by rdhabalia**
Create CI jobs for releases on branch-4.11.

### CODE REVIEW

File: .test-infra/jenkins/job_bookkeeper_release_branch_411_integrationtests.groovy [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+
+// This is the Java postcommit which runs a maven install, and the current set of precommit tests.
+freeStyleJob('bookkeeper_release_branch_48_integrationtests') {
```
Comment (eolivelli) [inline .test-infra/jenkins/job_bookkeeper_release_branch_411_integrationtests.groovy:22]: this name is wrong

File: .test-infra/jenkins/job_bookkeeper_release_branch_411_java8.groovy [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+
+// This job runs the Java postcommit tests on Java 8 for branch-4.8
+mavenJob('bookkeeper_release_branch_48_java8') {
```
Comment (eolivelli) [inline .test-infra/jenkins/job_bookkeeper_release_branch_411_java8.groovy:22]: this name is wrong

File: .test-infra/jenkins/job_bookkeeper_release_branch_411_java9.groovy [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+
+// This job runs the Java postcommit tests on Java 9 for branch-4.8
+mavenJob('bookkeeper_release_branch_48_java9') {
```
Comment (eolivelli) [inline .test-infra/jenkins/job_bookkeeper_release_branch_411_java9.groovy:22]: this name is wrong

**Comment by rdhabalia** [comment]
@eolivelli addressed the comments.

**Pull request merged**